collect_app/src/main/java/org/odk/collect/android/fragments/Camera2Fragment.java

```diff
@@ -82,15 +82,34 @@
     /**
      * Conversion from screen rotation to JPEG orientation.
      */
+
+    /*
+     * For front camera only.
+     * Bug fixed on 2018.2.17.
+     */
+
     private static final SparseIntArray ORIENTATIONS = new SparseIntArray();
     private static final int REQUEST_CAMERA_PERMISSION = 1;
     private static final String FRAGMENT_DIALOG = "dialog";
 
+    /**
+    * Fowling code only suits for back camera
+    *
+    * static {
+    * ORIENTATIONS.append(Surface.ROTATION_0, 90);
+    * ORIENTATIONS.append(Surface.ROTATION_90, 0);
+    * ORIENTATIONS.append(Surface.ROTATION_180, 270);
+    * ORIENTATIONS.append(Surface.ROTATION_270, 180);
+    * }
+    *
+    * For front camera, we use this:
+    * */
```

Comment:
I don’t see any value in this comment. The old values will be in the commit history. The doc comment is malformed and misplaced.